src/Nethermind/Nethermind.Consensus/Processing/ProcessingStats.cs

```diff
@@ -134,9 +134,17 @@ void IThreadPoolWorkItem.Execute()
 
             if (_lastBranchRoot is null || !_stateReader.HasStateForRoot(_lastBranchRoot) || block.StateRoot is null || !_stateReader.HasStateForRoot(block.StateRoot))
                 return;
-            UInt256 beforeBalance = _stateReader.GetBalance(_lastBranchRoot, beneficiary);
-            UInt256 afterBalance = _stateReader.GetBalance(block.StateRoot, beneficiary);
-            UInt256 rewards = beforeBalance < afterBalance ? afterBalance - beforeBalance : default;
+
+            UInt256 rewards = default;
+            try
+            {
+                UInt256 beforeBalance = _stateReader.GetBalance(_lastBranchRoot, beneficiary);
+                UInt256 afterBalance = _stateReader.GetBalance(block.StateRoot, beneficiary);
+                rewards = beforeBalance < afterBalance ? afterBalance - beforeBalance : default;
+            }
+            catch
+            {
+            }
```

Comment:
I think this catch should be on whole `Execute` method. and lets add logs on exception

Reply:
Added catch for whole Execute